server/api/region.go

```diff
@@ -99,6 +103,10 @@ func (h *regionHandler) GetRegion(w http.ResponseWriter, r *http.Request) {
 	}
 
 	regionInfo := rc.GetRegionByKey(paramsByte[0])
+	if regionInfo == nil {
+		h.rd.JSON(w, http.StatusNotFound, errs.ErrRegionNotFound.FastGenByArgs().Error())
```

Comment:
404 is expected?

Reply:
Which status code do you think is more reasonable?

Reply:
I was wondering if a 0 is valid request.

Reply:
why is 0 valid request?

Reply:
0 is invalid, but here you return 404 instead of 400.

Reply:
got it